.travis.yml

```diff
@@ -4,17 +4,25 @@ sudo: True
 language: 'python'
 python: '2.7'
 
+# https://github.com/koalaman/shellcheck/wiki/TravisCI
+addons:
+  apt:
+    sources:
+      - debian-sid
+    packages:
+      - shellcheck
```

Comment:
I don't think that this is a good idea. DebOps relies on a sane APT configuration for tests, if you mess it up by adding a repository from a different distribution, epseically Debian Sid, without taking care of removing that before the test, you will have a bad time. It can even be seen in the test that checked this commit.